tests/test_docker_compose.py

```diff
@@ -3,31 +3,40 @@
 from testcontainers.compose import DockerCompose
 from testcontainers.core.exceptions import NoSuchPortExposed
 from testcontainers.core.utils import inside_container
+from tests import TESTS_DIR
 
 
 def test_can_spawn_service_via_compose():
-    with DockerCompose("tests") as compose:
+    with DockerCompose(TESTS_DIR) as compose:
         host = compose.get_service_host("hub", 4444)
         port = compose.get_service_port("hub", 4444)
         assert host == "0.0.0.0"
         assert port == "4444"
 
 
 def test_can_pull_images_before_spawning_service_via_compose():
-    with DockerCompose("tests", pull=True) as compose:
+    with DockerCompose(TESTS_DIR, pull=True) as compose:
+        host = compose.get_service_host("hub", 4444)
+        port = compose.get_service_port("hub", 4444)
+        assert host == "0.0.0.0"
+        assert port == "4444"
+
+
+def test_can_build_images_before_spawning_service_via_compose():
+    with DockerCompose(TESTS_DIR, build=True) as compose:
```

Comment:
What does `docker-compose` do when `--build` is passed but the [`docker-compose.yml`](https://github.com/testcontainers/testcontainers-python/blob/master/tests/docker-compose.yml) file doesn't contain any reference to a locally-built image?

Reply:
Good question!

I just tried it with the `tests/docker-compose.yml` and it had no effect; it worked exactly as it would without it.

Reply:
@tillahoffmann :point_up: is this good enough?